cashctrl_ledger/ledger.py

```diff
@@ -47,11 +77,113 @@ def add_price():
         """
         raise NotImplementedError
 
-    def add_vat_code():
+    def add_vat_code(
+        self, code: str, rate: float, account: str, inclusive: bool = True,
+        text: str = ""
+    ):
         """
-        Not implemented yet
+        Adds a new VAT code to the CashCtrl account.
+
+        Parameters:
+            code (str): The VAT code to be added.
+            rate (float): The VAT rate, must be between 0 and 1.
+            account (str): The account identifier to which the VAT is applied.
+            inclusive (bool): Determines whether the VAT is calculated as 'NET' 
+                            (True, default) or 'GROSS' (False).
+            text (str): Additional text or description associated with the VAT code.
+
+        Raises:
+            ValueError: If any of the inputs are invalid (wrong type or out of allowed range).
+        """
+        accounts = self._client.list_accounts()
+        account_map = accounts.set_index('number')['id'].to_dict()
+        payload = {
+            "name": code,
+            "percentage": rate*100,
+            "accountId": account_map.get(account, None),
+            "calcType": "NET" if inclusive else "GROSS",
+            "documentName": text,
+        }
+
+        if not isinstance(payload['name'], str) or len(payload['name']) > 50:
+            raise ValueError(
+                "Invalid code. It must be a string with a maximum of 50 characters."
+            )
+
+        if not isinstance(payload['percentage'], (int, float)) \
+                or not (0.0 <= payload['percentage'] <= 100.0):
+            raise ValueError(
+                "Invalid rate. It must be a number between 0 and 1"
+            )
+
+        if not isinstance(payload['documentName'], str) \
+                or len(payload['documentName']) > 50:
+            raise ValueError(
+                "Invalid text. It must be a string with a maximum of 50 characters."
+            )
+
+        self._client.post("tax/create.json", data=payload)
+        
+    def update_vat_code(
```

Comment:
Please align Parameter description and code in `update_vat_code` with the latest version of `add_vat_code`.